### PR TITLE
ChatFragment: Improve it

### PR DESCRIPTION
**Pull request by uuilmer**
Was able to:

1. Make it so that the User can see new messages coming in Chats without needing to manually refresh.

Demonstration:

ezgif-6-939d84f67135

### CODE REVIEW

**Comment by uuilmer** [comment]
Was able to:

1. Combine consecutive messages into one if sent by the same User.

115979084_1305043269827717_1812339952017138020_n

**Comment by uuilmer** [comment]
Was able to improve the Snapchat theme of the ChatsFragment and ChatFragment:

115897520_742881536459598_3939716099538397714_n

115858212_292374162081685_2980232833286593916_n

File: app/src/main/java/com/example/pix/chat/ChatFragment.java [hunk 1]
```diff
@@ -191,6 +197,40 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView unusedView) {
...
+            new Timer().schedule(new TimerTask() {
+                @Override
+                public void run() {
+                    ParseQuery<Message> q = ParseQuery.getQuery(Message.class);
+                    q.whereEqualTo("chat", chat);
+                    q.whereEqualTo("to", ParseUser.getCurrentUser());
+                    // Check if there is a message in this Chat, to the current user who's time is greater
+                    // than our latest message
+                    q.whereGreaterThan("createdAt", lastMessage);
+                    q.orderByAscending("createdAt");
```
Comment (xuyueee) [inline app/src/main/java/com/example/pix/chat/ChatFragment.java]: String constants.

File: app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java [hunk 2]
```diff
@@ -66,36 +63,27 @@ public ViewHolder(@NonNull View itemView) {
...
-                if(content.getFrom().equals(ParseUser.getCurrentUser())){
-                    previous.text.setText("" + previous.text.getText().toString() + "\n" + message.getText());
-                    newPic(previous.contentPics, message);
-                    this.itemView.setVisibility(View.GONE);
-                    return;
+            Message message = messages.get(position);
+
+            // Check if the previous message was sent by the same person, if so there is no need to display the username again
+            if (position < messages.size()-1) {
+                if (messages.get(position+1).getFrom().getObjectId().equals(message.getFrom().getObjectId())){
```
Comment (xuyueee) [inline app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java]: Please add space before and after `-` and `+`.

Comment (xuyueee) [reply app/src/main/java/com/example/pix/chat/adapters/MessageAdapter.java]: And space before `{`.

**Comment by xuyueee** [comment]
Looks good. Feel free to merge.

**Pull request merged**